app/src/main/java/com/example/caregiver/ProfileInfo.java

```diff
@@ -83,26 +84,13 @@ public void onCreate(Bundle savedInstanceState) {
      * @param userId the user id of the currently logged in user
      */
     public void displayUserInfo(View view, String userId) {
-        DatabaseReference database = FirebaseDatabase.getInstance().getReference();
```

Comment:
I actually remove this part because we don't need to make another repeated call to the database when we have all the info in the Login page already.